internal/testmain/testmain_test.go

```diff
@@ -0,0 +1,213 @@
+package testmain_test
+
+import (
+	gobuild "go/build"
+	"go/token"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/gopherjs/gopherjs/build"
+	"github.com/gopherjs/gopherjs/internal/srctesting"
+	. "github.com/gopherjs/gopherjs/internal/testmain"
+)
+
+const inPackage = ``
+const external = ``
```

Comment:
I don't see these constants referenced anywhere. Am I loverlooking something?

Reply:
Oops,  that's a leftover from an early iteration. Removed.